.travis.yml

```diff
@@ -9,9 +9,7 @@ os:
 env:
   - "PATH=/home/travis/gopath/bin:$PATH"
 script:
-  - test -z "$(gofmt -l ./)"
-  - test -z "$(go vet -v ./...)"
-  - go build
```

Comment:
Travis *should* be running tests, IMO. The `gofmt` check in particular has already caught a couple misformatted PRs.